### PR TITLE
DTCPP-1578: Update to 1.3.4

### PR DESCRIPTION
**Pull request by sophialittlejohn**

### CODE REVIEW

File: src/index.js
Comment (sophialittlejohn) [inline src/index.js:153]: The only thing that broke while merging reach/router into our package was the automatic passing of the query in the LocationContext.Provider.

It's been fixed here and addtionally in the rasch-stack [PR 3337](https://github.com/rasch-dtc/rasch-stack/pull/3337)

**Review by JiiB — APPROVED** [review]
Looks good to me 👍🏻 

Make sure to merge this first, right after that you have to merge https://github.com/rasch-dtc/rasch-stack/pull/3329 to master (all devs need to be informed that the should update their working branches, otherwise a `yarn reinstall` will pull the latest router changes but the frontend for some devs might not be updated yet)

**Pull request merged**
